cclib/parser/gaussianparser.py

```diff
@@ -213,6 +216,229 @@ def extract(self, inputfile, line):
+        # 8.  C  0.0095  0.0115  1.0469  0.0121  0.0038  0.0079  0.0021  0.0000  0.7958
+        # 9.  C  0.0024  0.0089  0.7798  0.0096  0.0017  0.0037  0.0074  0.7958  0.0000
+        if 'Wiberg bond index matrix' in line:
+            WBO_matrix = []
+            while not 'Wiberg bond index,' in line:
+                line = inputfile.next()
+                if 'Atom' in line:
+                    line = inputfile.next()
+                    line = inputfile.next()
+                    WBO_block = []
+                    while len(line) > 1:
+                        WBO_block.append(line.split()[2:])
+                        line = inputfile.next()
+                    WBO_block = numpy.array(WBO_block)
+                    WBO_block = WBO_block.transpose().tolist()
+                    WBO_matrix += WBO_block
+            self.set_attribute('WBO_matrix', WBO_matrix)
+
+        # For TD calculations look for SCF energies of the first excited state:
+        # 
+        #  This state for optimization and/or second-order correction.
+        # Total Energy, E(TD-HF/TD-DFT) =  -149.392611305    
+        # Copying the excited state density for this state as the 1-particle RhoCI density.
+        elif 'E(TD-HF/TD-DFT)' in line:
+            if not hasattr(self, 'td_energy'):
+                self.td_energy = []
+            self.td_energy.append(utils.convertor(utils.float(line.strip().split()[-1]), "hartree", "eV"))
+            self.td_energy_fin = utils.convertor(utils.float(line.strip().split()[-1]), "hartree", "eV")
+            self.set_attribute('td_energy', self.td_energy)
+            self.set_attribute('td_energy_fin', self.td_energy_fin)
+
+        # For G4 calculations look for G4 energies (Gaussian16a bug prints G4(0 K) as DE(HF)) --Brian modified to work for G16c-where bug is fixed.
+        #
+        # Temperature=              298.150000 Pressure=                      1.000000
+        # E(ZPE)=                     0.098682 E(Thermal)=                    0.103138
+        # E(CCSD(T))=              -231.530419 E(Empiric)=                   -0.104205
+        # DE(Plus)=                  -0.014015 DE(2DF)=                      -0.182003
+        # E(Delta-G3XP)=             -0.339213 DE(HF)=                       -0.022842
+        # G4(0 K)=                 -232.094014 G4 Energy=                  -232.089558
+        # G4 Enthalpy=             -232.088614 G4 Free Energy=             -232.121498
+
+        elif line.strip().startswith('E(ZPE)='): #Overwrite DFT ZPE with G4 ZPE
+            self.zero_point_corr = float(line.strip().split()[1])
+        elif line.strip().startswith('G4(0 K)'):
+            self.G4_energy = float(line.strip().split()[2])
+            self.G4_energy -= self.zero_point_corr #Remove G4 ZPE
+            self.G4_energy = utils.convertor(self.G4_energy, "hartree", "eV")
+            self.set_attribute('g4_energy', self.G4_energy)
```

Comment:
Yes, here it is:

[benzene_g4.log](https://github.com/cclib/cclib/files/12497391/benzene_g4.log)

Reply:
OK, so currently the CCSD(T) energy part is parsed out correctly,  but not the final G4 energy I think. We'll need to decide where this should go (perhaps it should go in the existing `ccenergies` or maybe as a new property).

Reply:
IMO new property `composite_energies` or something along those lines. This would be of type dictionary with the label "G4" since there are various types of composite methods. Sound reasonable?

Reply:
sounds good

Reply:
sounds good to me too, but again might want to with `compositeenergies` or `compenergies` for consistency

Reply:
good point on the naming any of those sound good